html/controls/operator.js

```diff
@@ -2209,21 +2209,19 @@ function createSaveLoadContent() {
 	var sbDownloadTable = $("<table>").addClass("Download")
 		.appendTo($("<td>").appendTo($("<tr>").appendTo("#SaveLoad")));
 	$("<tr>").addClass("Name").appendTo(sbDownloadTable)
-		.append("<td colspan='4'>Download ScoreBoard XML</td>");
+		.append("<td colspan='4'>Download ScoreBoard JSON</td>");
 	$("<tr>").addClass("Instruction").appendTo(sbDownloadTable)
-		.append("<td colspan='4'>To download, right-click and Save - to view XML, left-click</td>");
+		.append("<td colspan='4'>To download, right-click and Save - to view JSON, left-click</td>");
 	var contentRow = $("<tr>").addClass("Content").appendTo(sbDownloadTable);
 
 	var links = [
 		{ name: "All data", url: "" },
-		{ name: "ScoreBoard", url: "scoreboard.xml?path=ScoreBoard" },
-		{ name: "Teams", url: "teams.xml?path=Teams" },
-		{ name: "Pages", url: "pages.xml?path=Pages" }
+		{ name: "Teams", url: "teams.json?path=ScoreBoard.PreparedTeam" }
```

Comment:
Should we add the game data file for the current game to this menu?

Reply:
Not a bad idea, but we'd probably want to re-think what counts as game-data first. The full scoreboard can serve the same purpose for now.